modules/home/transformations/base/transformations-interface.php

```diff
@@ -0,0 +1,13 @@
+<?php
+namespace Elementor\Modules\Home\Transformations\Base;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+interface Transformations_Interface {
```

Comment:
do we need this ?

Reply:
@TzviRabinovitch I have used it to make the `transform()` method required in all tranformations.

Reply:
looks like not necessary since we already have `abstract public function transform(): array;`